runtime/autoload/netrw.vim

```diff
@@ -283,7 +283,7 @@ if !exists("g:netrw_scp_cmd")
  if executable("scp")
   call s:NetrwInit("g:netrw_scp_cmd" , "scp -q")
  elseif executable("pscp")
-  if (has("win32") || has("win95") || has("win64") || has("win16")) && filereadable('c:\private.ppk')
+  if (has("win32")) && filereadable('c:\private.ppk')
```

Comment:
Could you change `(has("win32"))` to `has("win32")`?

Reply:
fixed 👌